src/gui/clips/SampleClipView.cpp

```diff
@@ -120,8 +119,7 @@ void SampleClipView::dropEvent( QDropEvent * _de )
 	}
 	else if( StringPairDrag::decodeKey( _de ) == "sampledata" )
 	{
-		m_clip->m_sampleBuffer->loadFromBase64(
-					StringPairDrag::decodeValue( _de ) );
+		auto buffer = SampleLoader::createBufferFromBase64(StringPairDrag::decodeValue(_de));
```

Comment:
```suggestion
		m_clip->setSampleBuffer(SampleLoader::createBufferFromBase64(StringPairDrag::decodeValue(_de)));
```
Looks like this wasn't setting the clip's sample buffer.